src/python/pants/backend/jvm/tasks/jvm_compile/compile_context.py

```diff
@@ -8,18 +8,20 @@
 import zipfile
 from contextlib import contextmanager
 
+from pants.backend.jvm.targets.jar_library import JarLibrary
 from pants.build_graph.aliased_target import AliasTarget
 from pants.build_graph.target import Target
 from pants.util.contextutil import open_zip
 
 
 def _resolve_strict_dependencies(target):
```

Comment:
And how do you like Nick's suggestion below as a quick tackle for this? I can create subsequent ticket if we want to implement strict_deps and exports for jar_library (which I am still not sure if that's necessary)